### ISSUE TITLE
certificate confirmation

### ISSUE THREAD

**Issue by MicZet** [dobin/lxd-webgui#40]
Hi!
It's not a bug - its more a question...

i setup everything like specified in documentation... and it works fine! 

but every time i create new session (so when I open the browser) it says
Could not get config from server...
and I need to go to settings 
Open LXD URL in new Tab
and confirm certificate

sorry if that's naive question - but is there a way to avoid confirmation of certificate every time?

regards,
Michael

**Comment by dobin** [comment]
Yes. Basically with #22 you create your own CA (certificate authority). Then you create the certificates based on this CA, and configure the browser to trust this CA. I'll update the readme with how to to it.
